### ISSUE TITLE
Ability to add middleware using broker config

### ISSUE THREAD

**Issue by mailaneel** [ice-services/moleculer#165]
When using moleculer runner, there is no easy way to provide the middleware using suggested `moleculer.config.js`

```js
module.exports = {
  nodeID: 'local-services',
  logger: true,
  logLevel: 'debug',
  middleware: [responseMiddleware()]
}
```

Without the above option, we still have to manually create broker and call `.use` for adding middleware.

**Comment by icebob** [comment]
Plus add `created` & `started` lifecycle handlers to. 
The `created` is called when the Runner created the ServiceBroker instance.
The `started` is called after the `broker.start()`.

**Comment by icebob** [comment]
Done. It will be released in v0.12

```js
// moleculer.config.js
module.exports = {
    logger: true,

    // Add middlewares
    middlewares: [myMiddleware()],

    // Fired when the broker is created
    created(broker) {
    },

    // Fired when the broker is started
    started(broker) {
        // You can return with Promise
        return broker.Promise.resolve();
    },

    // Fired when the broker is stopped
    stopped(broker) {
        // You can return with Promise
        return broker.Promise.resolve();
    }
};
```
